src/main/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConfig.java

```diff
@@ -41,6 +41,13 @@
 	public static final String DOCUMENT_MIMETYPE = "ml.document.mimeType";
 	public static final String DOCUMENT_URI_PREFIX = "ml.document.uriPrefix";
 	public static final String DOCUMENT_URI_SUFFIX = "ml.document.uriSuffix";
+	/*
+	 * v1.2.2 changes
+	 */
+	public static final String SSL = "ml.connection.ssl";
```

Comment:
I'd rename this to make it more clear that it's a boolean (in hindsight, I wish "ml.connection.simpleSsl" were "ml.connection.enableSimpleSsl"). I think "ml.connection.enableCustomSsl" would be a good descriptive name.

Reply:
Making the changes